src/System.Private.ServiceModel/tests/Common/Infrastructure/ConditionalTestDetectors.cs

```diff
@@ -142,6 +142,19 @@ public static bool IsWindows()
             return OSID.AnyWindows.MatchesCurrent();
         }
 
+        public static bool IslinuxWithIIS()
+        {
+            if (IsWindows())
+            {
+                return true;
+            }
+            else if (!IsIISHosted())
```

Comment:
true or false?

Reply:
I have update suitable name to 'IsNotLinuxWithIIS'.